src/components/button/button.scss

```diff
@@ -145,6 +145,11 @@ button.md-button::-moz-focus-inner {
     &.ng-hide, &.ng-leave {
       transition: none;
     }
+
+    md-icon {
+      // Fixes icon vertical alignment.
```

Comment:
I think this comment is actually not saying really much. 

Can we explain more, why this is needed and for example, which components are affected by the vertical alignment problem?